pandas/core/indexes/multi.py

```diff
@@ -740,32 +748,51 @@ def set_levels(self, levels, level=None, inplace=False, verify_integrity=True):
         Examples
         --------
         >>> idx = pd.MultiIndex.from_tuples([(1, 'one'), (1, 'two'),
-                                            (2, 'one'), (2, 'two')],
+                                            (2, 'one'), (2, 'two'),
+                                            (3, 'one'), (3, 'two')],
                                             names=['foo', 'bar'])
-        >>> idx.set_levels([['a', 'b'], [1, 2]])
+        >>> idx.set_levels([['a', 'b', 'c'], [1, 2]])
```

Comment:
Why do you think an extra value makes this example clearer? I think a more concise example is easier to read.

Reply:
My intention of adding an extra value at one of the levels (same for the _MultiIndex_ above) is to further highlight the differences in passing values to the two levels in the _MultiIndex_, for which the number of values in each level are not equal. Using different number of values in each level is to emphasize that _set_levels()_ passes values to indexes on 'by level' basis (by changing the _levels_ attribute), not on 'by row' basis.

Then again, if a more concise example could also explain sufficiently how _set_levels()_ work without ambiguity, I'm okay with reverting to the original _MultiIndex_.